include/inviwo/core/processors/canvasprocessor.h

```diff
@@ -92,6 +92,7 @@ class IVW_CORE_API CanvasProcessor : public Processor, public ProcessorWidgetMet
 public:
     ImageInport inport_;
 
+    CompositeProperty inputSize_;
```

Comment:
Will this not create a order warning in the constructor?

Reply:
Yes, fixed it